man/systemd-nspawn.xml

```diff
@@ -980,6 +980,20 @@
         effect.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>--notify-mode=</varname></term>
+
+        <listitem><para>Configures support for notifications from init process in the container.
+        <varname>--notify-mode=</varname> supports three options: <option>no</option>, <option>yes</option>, and <option>ready</option>.
+        Option <option>no</option> do not enable notifications between systemd-nspawn and
+        init process in the container and systemd-nspawn notifies systemd with a <literal>READY=1</literal>
+        message when the init process is created. Option <option>yes</option> enables notifications
+        but do nothing with them, systemd-nspawn notifies systemd with a <literal>READY=1</literal>
+        message when the init process is created. Option <option>ready</option>
+        enables notifications and wait for the <literal>READY=1</literal> message from
+        the init process in the container before sending its own to systemd.</para></listitem>
+      </varlistentry>
```

Comment:
I wonder what the benefits of the modes "no" and "yes" are. There's nothing wrong with always listening on the socket, is there?

I am tempted to say that we should always listen on the socket, and there should be a single boolean option `NotifyReady=` or so, that enables/disables whether nspawn shall wait for READY=1 from the init system in the container.

(Also, while we are at it, I think it would be good to forward the STATUS= text from the init system too, unconditionally...)